resources/assets/styles/components/_forms.scss

```diff
@@ -23,4 +23,10 @@ form {
   .search-form {
     width: 100%;
   }
+}
+
+.bkgd-blue-navy {
+  .form-group {
+    margin-bottom: 0;
```

Comment:
this puts control for margins in two places (component with class `mb-3` and css style) - wondering if this sets us up for confusion down the road?